index.md

```diff
@@ -59,9 +59,11 @@ Non-public contact via email: [organizers email][contact]
 <div class="medium-4 columns">
 ### Upcoming
 
-* Wednesday, February 26th 2025
+[Calendar subscription link](https://emacs-berlin.org/calendar.ics)
+
+* Tuesday, March 11th 2025
 * Wednesday, March 26th 2025
-* Wednesday, April 30th 2025
```

Comment:
This line should not get deleted but moved down into the commented out list instead. I'm using that commented out list to pick the next date when updating the page without having to think too much.